starters/gatsby-wordpress-starter/gatsby-node.js

```diff
@@ -112,81 +113,49 @@ async function createPostArchive({ posts, gatsbyUtilities }) {
 async function createPostIndex({ posts, gatsbyUtilities, routing }) {
 	const itemsPerPage = 12
 
-	const postsChunkedIntoArchivePages = chunk(posts, itemsPerPage)
+	const totalPages = Math.ceil(posts.length / itemsPerPage)
 
-	return Promise.all(
-		postsChunkedIntoArchivePages.map(async (_posts, index) => {
-			const pageNumber = index + 1
-
-			const getPagePath = page => {
-				return `/posts/${page}`
-			}
-
-			if (index === 0 && routing === true) {
-				await gatsbyUtilities.actions.createPage({
-					path: '/posts',
-
-					component: path.resolve(`./src/templates/postsIndex.jsx`),
-
-					context: {
-						itemsPerPage,
-						routing,
-						breakpoints: { start: 4, end: 6, add: 2 },
-					},
-				})
-			}
+	const getPagePath = page => {
+		return page === 0 ? '/posts' : `/posts/${page}`
+	}
```

Comment:
You can make this reusable in such a way that you don't have to write it twice (line 142)